pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -204,3 +204,23 @@ func isAntreaPolicyEgressTable(tableID uint8) bool {
 	}
 	return false
 }
+
+// getPacketInTableID returns the OVS table ID in which the packet is sent to antrea-agent. Since L2ForwardOutput is
+// the table where all Antrea-native policies logging packets are sent to antrea-agent, "PacketInTableField" is used
+// to store the real table requiring "sendToController" action. This function firstly parses the direct table where
+// the packet leaves OVS pipeline, then checks whether "PacketInTableField" is set with a valid value or not. The value
+// in the field is returned if yes.
+func getPacketInTableID(pktIn *ofctrl.PacketIn) uint8 {
```

Comment:
Having offline sync with @tnqn , since we are recording the OF table name in an audit log entry, I will continue to load the table ID in a packetIn message via a separate OVS register field. This logic may be updated in future after refining the audit logging format.